README.md

```diff
@@ -1,48 +1,105 @@
-# Snowflake Native SDK for Connectors
+# Native SDK for Connectors
```

Comment:
In docs we use: Snowflake Native SDK for Connectors so I would leave it as it is